tests/models/gptj/test_modeling_gptj.py

```diff
@@ -518,6 +527,48 @@ def test_model_from_pretrained(self):
             model = GPTJModel.from_pretrained(model_name, revision="float16", torch_dtype=torch.float16)
             self.assertIsNotNone(model)
 
+    @require_flash_attn
+    @require_torch_gpu
+    @pytest.mark.flash_attn_test
+    @slow
+    def test_flash_attn_2_generate_padding_right(self):
```

Comment:
requires_bitsandbytes here! 
Also let's add the expected text explicitly! to make sure we always have what we want!